pkg/snapshot/restorer/types.go

```diff
@@ -33,6 +33,9 @@ const (
 	defaultInitialAdvertisePeerURLs = "http://localhost:2380"
 	defaultInitialClusterToken      = "etcd-cluster"
 	defaultMaxFetchers              = 6
+	defaultMaxCallSendMsgSize       = 10 * 1024 * 1024       //10Mib
+	defaultMaxRequestBytes          = 10 * 1024 * 1024       //10Mib
+	defaultMaxTxnOps                = 10 * 1024              //10Kib
```

Comment:
As this value does not represent a size but simply the amount of operations, I think I would avoid the `Kib` comment.

Question to me would also be if it would make sense to chop the operations in `applyEventsToEtcd` into smaller batches in case it exceeds the `maxTxnOps` value. Wouldn't this prevent this kind of failure in the first place? But this would be a separate issue.

Reply:
Hi @Gerrit91 Yes, I would remove the comment shortly.. 

Yeah, I am currently working on this.. So far I have seen that `check perf` or any large operations are transacting in a particular order which is okay with the Etcd cluster.. But after we are taking delta snapshot, we are losing that orders probably. I am trying to look for a way to preserve the same order and if not possible, then I will do something as you suggested.. I would like to raise that as another PR.